### ISSUE TITLE
Powermax Pro lose connection con Home Assistant

### ISSUE THREAD

**Issue by robermf** [davesmeghead/visonic#63]

**Comment by robermf** [comment]
Hello!
After a few years running perfect I have found this problem: 

Home Assistant lose the connection with the panel. I have deleted everything, even trying in fresh install Home Assistant. Also tried a new TTL to ethernet converter. And reset the panel (even removing AC and battery)
The connection is in Powerlink mode. 

Here is the log: https://pastebin.com/TFCUq8v4

Forcing Standard Mode I get the same. 

Any ideas about what can I tried? Thanks!!

**Comment by davesmeghead** [comment]
I'm not certain what is going on with your overall setup but I can tell you from looking at the log file that the communication link between the panel and the integration keeps failing and recovering over and over. The log file indicates that everything is working and in Powerlink mode but then communication stops for several minutes and then is back to being OK again.  In the Integration there are timeouts to indicate that there is a problem with the connection to the panel.

If you have tried the panel and TTL gadget then is there are different way that you can make the connection, bypass your router or perhaps use USB?
Does it happen straight away (and therefore easy to test) or does it take hours before it does this?

**Comment by robermf** [comment]
SOLVED!!!!!

Dave, first of all, thank you for your help. 

And now, the problem: IT WAS THE SIM CARD. 


About 10 days before I found the problem (I guess began before that 10 days) I made some changes:

-Change location of the panel. 
-Move the buzzer to another location.
-Move the speaker to another location (don't do this, the sound it is soooo quiet)
-Change the SIM card to another carrier.


Ok. Last week I tried almost everything: Check the internals, back buzzer and speaker, try another GSM module, re install integration, HA fresh installation, use another TTL to Ethernet adapter....

But not, was the SIM card. The last thing I tried (I made tests without GSM module, later with the module without SIM...). 5 years ago I was using the Visonic app, and I configured the APN of my carrier, and the IP of the server. After your awesome integration I disabled GPRS send but.... THAT WAS THE CAUSE.

Now with the new SIM and after delete APN and the IP it looks all is fine. A tons of hours spent but it is OK when you get a happy ending. 

Again, thank you Dave. I am going to write you in HA community. I am the guy who asked about add two panels to HA. 


Have a great day!
